py/desitarget/QA.py

```diff
@@ -28,7 +28,7 @@
 from desiutil import brick
 from desiutil.log import get_logger, DEBUG
 from desiutil.plots import init_sky, plot_sky_binned
-from desitarget import desi_mask
+from desitarget.targetmask import desi_mask
```

Comment:
Is there a particular reason why you're doing
```python
from desitarget.targetmask import desi_mask
```
instead of 
```python
from .targetmask import desi_mask
```